app/Console/Commands/BibleEquivalents/UpdateBibleLinkOrganizations.php

```diff
@@ -55,14 +54,23 @@ public function handle()
             }
 
             // Otherwise Fuzzy Search for Provider Name
-            $organizations = Searchy::search('dbp.organization_translations')->fields('name')->query($bible_link->provider)->getQuery()->limit(5)->get();
-            if ($organizations->count() == 0) {
+            $organizatios = OrganizationTranslation::whereFuzzy('name', $bible_link->provider)
```

Comment:
typo... should be organizations

Reply:
It has been fixed in the next PR:
- https://github.com/faithcomesbyhearing/dbp/pull/474